docs/spec.md

```diff
@@ -415,9 +415,9 @@ tensor. Depending on the element type, does the following:
 
```

Comment:
Please update the PR description to follow the testing guidelines, e.g. similarly to #1080.